### ISSUE TITLE
demo_guizmo_sequencer at hidpi: GUI is a mess

### ISSUE THREAD

**Issue by dcnieho** [pthom/imgui_bundle#25]
Placement is not taking some scaling into account and everything overlaps or e.g. input boxes for frame atop the screen are too small, see here:
![image](https://user-images.githubusercontent.com/1787673/205750069-d517ac49-db6b-4b31-bcb5-b11bcbd0d809.png)

**Comment by pthom** [comment]
Thanks for reporting.

This issue lies in ImGuizmo itself, and happens only in the C++ demos, since I did not port ImSequencer to python (its api uses `int **` and I did not want to adapt these).

**Comment by pthom** [comment]
I vote for disabling this demo.

**Comment by district10** [comment]
@pthom unable to checkout submodules from main:

```
Unable to checkout '9e5061cfe09fabfaa789ea5a1591c9899f4e8f79' in submodule path 'external/ImGuizmo/ImGuizmo'
Unable to checkout '9aa1b4307780b42e706c189781cadbc6e05bcd90' in submodule path 'external/imgui_tex_inspect'
```

https://github.com/CedricGuillemet/ImGuizmo/tree/9e5061cfe09fabfaa789ea5a1591c9899f4e8f79

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

**Comment by pthom** [comment]
> @pthom unable to checkout submodules from main:
> 
> ```
> Unable to checkout '9e5061cfe09fabfaa789ea5a1591c9899f4e8f79' in submodule path 'external/ImGuizmo/ImGuizmo'
> Unable to checkout '9aa1b4307780b42e706c189781cadbc6e05bcd90' in submodule path 'external/imgui_tex_inspect'
> ```

Hum, I reviewed quite a lot the external libraries. Many are now using a fork on my account.

I think you were trying to update an existing checkout (It does work on a fresh checkout, I just tested thisà.

In this case, you can either:

````
git submodule sync
````

or remove imgui_bundle/external and reupdate it:

````
cd imgui_bundle
rm -rf external/
git pull
git submodule update --init --recursive
````

**Comment by district10** [comment]
@pthom 
I guess you updated your own .gitmodule remote url (or .git/config), current main branch of imgui_bundle is not working.

See <https://github.com/pthom/imgui_bundle/tree/main/external/ImGuizmo>.

**Comment by pthom** [comment]
> I guess you updated your own .gitmodule remote url (or .git/config), current main branch of imgui_bundle is not working.

You were right, sorry for this. This is now fixed

**Comment by district10** [comment]
@pthom Still need to fix `imgui_tex_inspect @ 9aa1b43` in <https://github.com/pthom/imgui_bundle/tree/main/external>.

**Comment by pthom** [comment]
> Still need to fix imgui_tex_inspect

Fixed. Sorry, git submodule is quickly clumsy when you have a lot of them, and try to change their url.
